lib/templates/pobject/system.svn.info.php

```diff
@@ -6,7 +6,7 @@
 		$fstore	= $this->store->get_filestore_svn("templates");
 		$svn	= $fstore->connect($this->id);
 		$svn_info = $fstore->svn_info($svn);
-		if( is_array($svn_info) ) {
+		if(isset($svn_info) && count($svn_info)) {
```

Comment:
why change to isset? This opens the way for a non-array type being passed to count()